src/stylesheets/_typography.scss

```diff
@@ -19,6 +20,33 @@ $font-family-italic: "Montserrat Italic", sans-serif;
   }
 }
 
+@mixin scheduleheader-text {
```

Comment:
Because this text styling will be used in multiple places, can we use a more generic name and add color variations within the SCSS file itself? This will help reduce how repetitive our styles can get.